### ISSUE TITLE
Improve visual design / engagement

### ISSUE THREAD

**Issue by toddmedema** [toddmedema/electrify#45]
Right now it's pretty much "spreadsheet the game". Would love any help / ideas on how to make it more visually interesting to appeal to a more mainstream audience!

**Comment by n-hebert** [comment]
This might be a bucket-ticket or project, it's likely worth spinning out sub-tickets to discuss individual items that sound worth having.

As an "easy to say, hard to do" big one, I would suggest an overhaul which eliminates the "Learn to play" mode entirely. These should just be the first missions. Ready for my opinionated take? :smiley: 

As a life-long gamer I found it kind of hard to immediately tell the mapping between the scenarios and the tutorials. I would ideally want to jump in feet first and learn on the ground, but I'm immediately faced with a choice I don't understand. When I regress to enter the tutorial, it's a lot of pop-up boxes that don't let me test my knowledge. There's currently a heavy weight of "tell don't play" which needs to be reversed to "play don't tell". A way that some game designers really make head-way in this space is to imagine that someone said "We need to ship this to [foreign country] next week and there's no budget to localize any text. Go." Knowing that you can't use any text, you'd start to swap out for symbols. Rendering pictorial/ideographic game prompts for any and all textual representations and slowly training on the meaning of those symbols is the means of eliminating these text prompts.

Not an easy idea, but I believe a low barrier to entry is critical for any game looking to get off the ground.

**Comment by toddmedema** [comment]
Yes, absolutely, that's the right mindset! And that's a very good point, the best games I've played never ask you to pick, they teach you as you go.

(also agree that this ticket is just a discussion starter to drive more specific actionable tickets)
